third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_rocm.tpl

```diff
@@ -242,9 +243,11 @@ def main():
     gpu_linker_flags.append('-L' + ROCR_RUNTIME_PATH)
     gpu_linker_flags.append('-Wl,-rpath=' + ROCR_RUNTIME_PATH)
     gpu_linker_flags.append('-l' + ROCR_RUNTIME_LIBRARY)
-    gpu_linker_flags.append('-L' + HCC_RUNTIME_PATH)
-    gpu_linker_flags.append('-Wl,-rpath=' + HCC_RUNTIME_PATH)
-    gpu_linker_flags.append('-l' + HCC_RUNTIME_LIBRARY)
+    # do not link with HCC runtime library in case hip-clang toolchain is used
+    if not HIPCC_IS_HIPCLANG:
+      gpu_linker_flags.append('-L' + HCC_RUNTIME_PATH)
+      gpu_linker_flags.append('-Wl,-rpath=' + HCC_RUNTIME_PATH)
+      gpu_linker_flags.append('-l' + HCC_RUNTIME_LIBRARY)
```

Comment:
@whchung 
Can we add the control statement "else" here since we also no need to include "hip-clang" related flags when "hcc" is used for compilation, I guess?
@yxsamliu Could you please confirm the above?

Reply:
we must link with HIP runtime library

Reply:
> @whchung
> Can we add the control statement "else" here since we also no need to include "hip-clang" related flags when "hcc" is used for compilation, I guess?
> @yxsamliu Could you please confirm the above?

We do not have hip-clang specific linker flags.